testcases/pkcs11/sess_mgmt.c

```diff
@@ -1197,6 +1197,171 @@ CK_RV do_OperationState3(void)
     return rc;
 }
 
+CK_RV do_SessionCancel(void)
+{
+    CK_SLOT_ID slot_id;
+    CK_SESSION_HANDLE session1;
+    CK_FLAGS flags;
+    CK_BYTE user_pin[PKCS11_MAX_PIN_LEN];
+    CK_ULONG user_pin_len;
+    CK_RV rc;
+
+    CK_BYTE original[1024];
+    CK_BYTE crypt1[1024];
+    CK_BYTE crypt2[1024];
+    CK_BYTE trash1[8];
+    CK_BYTE trash2[8];
+
+    CK_ULONG orig_len, crypt1_len, crypt2_len, trash1_len, trash2_len;
+    CK_ULONG i;
+
+    CK_ULONG key_len = 16;
+    CK_ATTRIBUTE key_gen_tmpl[] = {
+        {CKA_VALUE_LEN, &key_len, sizeof(CK_ULONG)}
+    };
+
+    CK_MECHANISM mech;
+    CK_OBJECT_HANDLE h_key;
+
+
+    printf("do_SessionCancel...\n");
+    slot_id = SLOT_ID;
```

Comment:
TRACE_INFO instead of printf ? And maybe remove one of the two blank lines ...

Reply:
The whole sess_mgmt testcase does not use the regular testcase_begin, etc macros from regress.h....
I would like to change it to use that at some point in time, but not in this PR. 
The other functions also use printf at that places, so lets leave it as is for now, and adapt in globally at a later time.

I will remove the blank line.